src/components/results/Results.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import ResultsHeader from './resultsComponents/ResultsHeader';
+import ResultsBody from './resultsComponents/ResultsBody';
+
+class Results extends Component {
+	render() {
+		var bookedItems = this.props.FakeBookings;
+
+		return (
+			<div>
+				<div> Search Reasults </div>
+				<table className="table">
+					<ResultsHeader bookings={bookedItems} />
```

Comment:
Not a huge problem, but it looks like you're not using the `bookings` prop in `ResultsHeader` so I wouldn't pass it